packages/flutter/lib/src/material/chip_theme.dart

```diff
@@ -0,0 +1,425 @@
+  final EdgeInsetsGeometry padding;
+
+  /// The border to draw around the chip.
+  ///
+  /// Defaults to a [StadiumBorder]. Must not be null.
+  final ShapeBorder shape;
+
+  /// The style to be applied to the chip's label.
+  ///
+  /// This only has an effect on label widgets that respect the
+  /// [DefaultTextStyle], such as [Text].
+  final TextStyle labelStyle;
+
+  /// An alternate style to be applied to the chip's label. For example, this
+  /// style is applied to the text of a selected [ChoiceChip].
+  ///
+  /// This only has an effect on label widgets that respect the
+  /// [DefaultTextStyle], such as [Text].
+  final TextStyle secondaryLabelStyle;
+
+  /// The brightness setting for this theme.
+  ///
+  /// This affects various base material color choices in the chip rendering.
+  final Brightness brightness;
+
+  /// Creates a copy of this object but with the given fields replaced with the
+  /// new values.
+  ChipThemeData copyWith({
+    Color backgroundColor,
+    Color deleteIconColor,
+    Color disabledColor,
+    Color selectedColor,
+    Color secondarySelectedColor,
+    EdgeInsetsGeometry labelPadding,
+    EdgeInsetsGeometry padding,
+    ShapeBorder shape,
+    String deleteButtonTooltipMessage,
+    TextStyle labelStyle,
+    TextStyle secondaryLabelStyle,
+    Brightness brightness,
+  }) {
+    return new ChipThemeData(
+      labelStyle: labelStyle ?? this.labelStyle,
+      secondaryLabelStyle: secondaryLabelStyle ?? this.secondaryLabelStyle,
+      shape: shape ?? this.shape,
+      backgroundColor: backgroundColor ?? this.backgroundColor,
+      padding: padding ?? this.padding,
+      labelPadding: labelPadding ?? this.labelPadding,
+      deleteIconColor: deleteIconColor ?? this.deleteIconColor,
+      selectedColor: selectedColor ?? this.selectedColor,
+      secondarySelectedColor: secondarySelectedColor ?? this.secondarySelectedColor,
+      disabledColor: disabledColor ?? this.disabledColor,
+      brightness: brightness ?? this.brightness,
+    );
+  }
+
+  /// Linearly interpolate between two chip themes.
+  ///
+  /// The arguments must not be null.
+  ///
+  /// The `t` argument represents position on the timeline, with 0.0 meaning
+  /// that the interpolation has not started, returning `a` (or something
+  /// equivalent to `a`), 1.0 meaning that the interpolation has finished,
+  /// returning `b` (or something equivalent to `b`), and values in between
+  /// meaning that the interpolation is at the relevant point on the timeline
+  /// between `a` and `b`. The interpolation can be extrapolated beyond 0.0 and
+  /// 1.0, so negative values and values greater than 1.0 are valid (and can
+  /// easily be generated by curves such as [Curves.elasticInOut]).
+  ///
+  /// Values for `t` are usually obtained from an [Animation<double>], such as
+  /// an [AnimationController].
+  static ChipThemeData lerp(ChipThemeData a, ChipThemeData b, double t) {
+    assert(a != null);
```

Comment:
I don't think this is necessary. Most (all?) lerp methods allow the `a` and `b` parameters to be null.